modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/resource/ResourceManagerBean.java

```diff
@@ -764,6 +766,10 @@ private boolean uninventoryResourceBulkDeleteAsyncWork(Subject overlord, int res
 
         boolean hasErrors = false;
         boolean debugEnabled = LOG.isDebugEnabled();
+
+        // Get all packages installed in this resource
```

Comment:
I'd like to see more detail in this comment to explain why this is different than everything else: